### ISSUE TITLE
VAMC Facility name change:  Wenatchee VA Clinic -> Elwood “Bud” Link Department of Veterans Affairs Outpatient Clinic [Awaiting redirect]

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#13821]
Redirect request: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13829

## Backstory:
This facility name change first came to us in May 2023 mistakenly applied to the Fort Belvoir VA Clinic in _VA Washington DC_ instead of VA Spokane _in Washington State._ The Fort Belvoir VA Clinic's name has since been changed back.

On 5/31/23, the Facilities API had flagged that the 'Wenatchee VA Clinic' in VA Spokane had been renamed to the 'Elwood “Bud” Link Department of Veteran Affairs Outpatient Clinic'. This is correct!

Instead of opening a brand new ticket, I am re-opening this one since it's already associated with this name.

## Intake
- [x] What triggered this runbook? (Flag in CMS, Help desk ticket, Product team, VHA Digital Media)
Trigger: Flagged content view

- [x] Link to associated help desk ticket (if applicable)
Help desk ticket: N/A

- [x] Name of submitter (if applicable)
Submitter: @jilladams 

- [x] If the submitter is an editor, send them links to any relevant KB articles for the VAMC product. Let them know that facility changes can take between 75 days and 4 months after submitting a request, according to VAST administrators.
KB article: ["How Do I Update My VAMC Facility's Basic Location Data?"](https://prod.cms.va.gov/help/vamc/how-do-i-update-my-vamc-facilitys-basic-location-data)

- [x] Link to facility in production:
Facility CMS link:  https://prod.cms.va.gov/spokane-health-care/locations/wenatchee-va-clinic
Facility API ID:  vha_668GA

#### CMS help desk steps
**Note: If the help desk is waiting on information from the facility staff or editor, add the "Awaiting editor" flag to the facility with a log message that includes a link to this ticket. Remove the flag when the ticket is ready to be worked by the Facilities team. Be sure to preserve the current moderation state of the node when adding or removing the flag.**
What happens: The name change is made in VAST, that syncs to Lighthouse which syncs to Drupal.
- [x] 1. Check that the title change in name field on the VAMC node has shown up in Drupal.
- [x] 2. Create a [URL change request](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&template=runbook-facility-url-change.md&title=URL+Change+for%3A+%3Cinsert+facility+name%3E), changing the entry from the old facility URL to the new facility URL. (**Note: The URL change request ticket blocks the completion of this ticket.**)

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13829

#### CMS Engineer steps
- [ ] 3. Execute the steps of the URL change request ticket from step 2 above.

(Redirects deploy weekly on Wed. at 10am ET, or by requesting OOB deploy (of the revproxy job to prod) in #vfs-platform-support. Coordinate the following items below and canonical URL change after URL change ticket is merged, deployed, and verified in prod.)

#### Drupal Admin steps (CMS Engineer or Help desk)
_Help desk will complete these steps or escalate to request help from CMS engineering._
- [x] 4. Locate the newly renamed VAMC Facility (https://prod.cms.va.gov/admin/content/bulk) Search by new name
- [x] 5. Updates URL alias for this facility
- [x] 6. Resave this facility
- [x] 7. Make bulk alias changes to facility service nodes. (https://prod.cms.va.gov/admin/content/bulk?type=health_care_local_health_service)
- [x] 8. Bulk save fixed titles to facility service nodes. (https://prod.cms.va.gov/admin/content/bulk?type=health_care_local_health_service)
- [x] 9. Update menu title for facility
- [x] 10. May also need to directly edit the VAMC System menu to alpha sort the menu item after the title changes
- [x] 11. Update Alt text for facility image, if relevant
- [x] 12. Update Meta description (TBD: some backwards compatibility for SEM, by including something like ", formerly known as [previous name]".
- [x] 13. Edit facility node and remove flag `Changed name` then save node (with moderation state = published)

#### CMS Help desk (wrap up)
- [ ] 14. Notify editor and any other stakeholders

**Comment by stefaniefgray** [comment]
@jilladams This looks good to go, thanks for creating! 

Redirect request: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13829

**Comment by stefaniefgray** [comment]
CLOSING

Clinic should actually remain Fort Belvoir per https://va-gov.atlassian.net/browse/VAHELP-5856

@omahane Is there anything else we can do on our end regarding a false name change? I told the editor to contact their VAST administrator ASAP but it seems like that's all we can do

**Comment by stefaniefgray** [comment]
Re-opening -- see notes at top of ticket

**Comment by stefaniefgray** [comment]
@jilladams @kmariepat-cityfriends New facility name change ticket + linked redirect request: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13829

**Comment by omahane** [comment]
@stefaniefgray @TroyA6 This has been updated and is ready for stakeholder confirmation.

**Comment by stefaniefgray** [comment]
Waiting on redirect

**Comment by stefaniefgray** [comment]
Waiting on VAST error in linked redirect ticket to be resolved

**Comment by omahane** [comment]
The facility name, link and URL are updated on prod. We can close this ticket after stakeholder confirmation, @stefaniefgray @TroyA6

**Comment by stefaniefgray** [comment]
Facility Locator still directs to Wenatchee CMS link as of 7/25/23

**Comment by omahane** [comment]
The CSV has been deployed.

**Comment by stefaniefgray** [comment]
Complete! Closing -- thanks @omahane 🚀

**Comment by stefaniefgray** [comment]
Notified editor in https://va-gov.atlassian.net/browse/VAHELP-6189
